examples/tutorials/analysis-1d/ebl.py

```diff
@@ -0,0 +1,292 @@
+# H.E.S.S. when it was in a steady state. The data have already been
+# reduced to OGIP format\ `SpectrumDatasetOnOff` following the procedure
+# :doc:`/tutorials/analysis-1d/spectral_analysis` tutorial using a
+# `ReflectedRegions` background estimation. The spectra and IRF from the
+# 6 observations have been stacked together.
+#
+# We will load this dataset as a `SpectrumDatasetOnOff` and proceed with
+# the modeling. You can do a 3D analysis as well.
+#
+
+dataset = SpectrumDatasetOnOff.read(
+    "$GAMMAPY_DATA/PKS2155-steady/pks2155-304_steady.fits.gz"
+)
+
+print(dataset)
+
+
+######################################################################
+# Model the observed spectrum
+# ---------------------------
+#
+# The observed spectrum is already attenuated due to the EBL. Assuming
+# that the intrinsic spectrum is a power law, the observed spectrum is a
+# `CompoundSpectrumModel` given by the product of an EBL model with the
+# intrinsic model.
+#
+
+
+######################################################################
+# For a list of available models, see
+# :doc:`/api/gammapy.modeling.models.EBL_DATA_BUILTIN`.
+#
+
+print(EBL_DATA_BUILTIN.keys())
+
+# define the power law
+index = 2.3
+amplitude = 1.81 * 1e-12 * u.Unit("cm-2 s-1 TeV-1")
+reference = 1 * u.TeV
+pwl = PowerLawSpectralModel(index=index, amplitude=amplitude, reference=reference)
+pwl.index.frozen = False
+# Specify the redshift of the source
+redshift = 0.116
+
+# Load the EBL model. Here we use the model from Dominguez, 2011
+absorption = EBLAbsorptionNormSpectralModel.read_builtin("dominguez", redshift=redshift)
+
+
+# The power-law model is multiplied by the EBL to get the final model
+spectral_model = pwl * absorption
+print(spectral_model)
+
+# Now, create a sky model and proceed with the fit
+sky_model = SkyModel(spatial_model=None, spectral_model=spectral_model, name="pks2155")
+
+dataset.models = sky_model
+
+fit = Fit()
+result = fit.run(datasets=[dataset])
+
+# we make a copy here to compare it later
+model_best = sky_model.copy()
+
+print(result.models.to_parameters_table())
+
+
+######################################################################
+# Get the flux points
+# ===================
+#
+# To get the observed flux points, just run the `FluxPointsEstimator`
+# normally
+#
+
+energy_edges = dataset.counts.geom.axes["energy"].edges
+fpe = FluxPointsEstimator(
+    energy_edges=energy_edges, source="pks2155", selection_optional="all"
+)
+flux_points_obs = fpe.run(datasets=[dataset])
+
+
+######################################################################
+# To get the deabsorbed flux points (ie, intrinsic points), we simply need
+# to set the reference model to the best fit power law instead of the
+# compound model. We first make a copy of the computed flux points
+#
+
+flux_points_intrinsic = deepcopy(flux_points_obs)
+flux_points_intrinsic._reference_model = SkyModel(spectral_model=pwl)
+
+print(flux_points_obs._reference_model)
```

Comment:
```suggestion
print(flux_points_obs.reference_model)
```
Can't you directly call the property `reference_model` itself?